lib/i18n/payment-method-service.ts

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
+
+import { PAYMENT_METHOD_SERVICE } from '../constants/payment-methods';
+
+const messages = defineMessages({
+  [PAYMENT_METHOD_SERVICE.BANK]: {
+    id: 'paymentMethodService.bank',
+    defaultMessage: 'Bank Account',
+  },
+});
+
+export const i18nPaymentMethodService = (service, intl) => {
+  const PaymentMethodServiceI18n = {
+    [PAYMENT_METHOD_SERVICE.PAYPAL]: 'PayPal',
+    [PAYMENT_METHOD_SERVICE.WISE]: 'Wise',
+    [PAYMENT_METHOD_SERVICE.BANK]: intl.formatMessage(messages[PAYMENT_METHOD_SERVICE.BANK]),
```

Comment:
Not an option for me, it's nothing/unknown/other which will be noted as an empty paymentMethod on the backend side.

Reply:
I'll remove this.